### PR TITLE
Removes the Extend-o-Hand

### PR DESCRIPTION
**Pull request by Iamgoofball**
1. attack_hand on 90% of objects was designed with "you're next to the machine" in mind
2. this is shit code, doesnt call ..(), and has long range(ha) balance implications and fucks shit up bigtime
3. doesnt even work properly and bypasses all normal shit for calling punches/disarms/etc.
4. we had huge arguments over changeling tentacles and curator whips disarming people and we allowed this?

### CODE REVIEW

**Comment by deathride58** [comment]
Needs `i ded` label

**Comment by kevinz000** [comment]
honestly this is one of those things that are neat but should have better code

**Comment by Spaceinaba** [comment]
fix the code then dork

**Comment by Iamgoofball** [comment]
@Spaceinaba the proper fix for this would require a recode of how attacking and clicking works and I'm not about that life

**Comment by Armhulen** [comment]
this is sadly correct

**Comment by PKPenguin321** [comment]
I whipped this up in like 30 seconds so I don't really mind if this goes through
Just curious what exactly prompted this? Did they turn out to be OP or something

**Comment by BeeSting12** [comment]
@PKPenguin321 
1. attack_hand on 90% of objects was designed with "you're next to the machine" in mind
2. this is shit code, doesnt call ..(), and has long range(ha) balance implications and fucks shit up bigtime
3. doesnt even work properly and bypasses all normal shit for calling punches/disarms/etc.
4. we had huge arguments over changeling tentacles and curator whips disarming people and we allowed this?

**Comment by PKPenguin321** [comment]
no yeah i read that but i mean those are just kind of existing conditions, usually PRs like these are prompted by something out of the ordinary happening in game

**Comment by 81Denton** [comment]
@PKPenguin321 Some monk killed someone with punches that didn't show messages or something

**Comment by Iamgoofball** [comment]
Which prompted me to check the code out looking for a bugfix and then oh god

**Comment by KorPhaeron** [comment]
If extendo hands dont function it probably means reach weapons in general don't function, and if you're going to remove the only reach weapon for being non functional you may as well remove the rest of reach code as well

**Comment by Iamgoofball** [comment]
@KorPhaeron nono, it's not an issue with reach code

lemme just code comment to explain the issue

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -614,26 +614,3 @@
...
-/obj/item/extendohand/acme
-	name = "\improper ACME Extendo-Hand"
-	desc = "A novelty extendo-hand produced by the ACME corporation. Originally designed to knock out roadrunners."
-
-/obj/item/extendohand/attack(atom/M, mob/living/carbon/human/user)
-	var/dist = get_dist(M, user)
-	if(dist < reach)
-		to_chat(user, "<span class='warning'>[M] is too close to use [src] on.</span>")
-		return
-	M.attack_hand(user)
```
Comment (Iamgoofball) [inline code/game/objects/items/weaponry.dm]: see this fucker right here? it's the issue

**Review by Cyberboss — APPROVED** [review]

**Comment by KorPhaeron** [comment]
@Iamgoofball in that case just the icons then please

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Icons

**Comment by Qbopper** [comment]
just make it admin spawn only because it's a funny meme

**Comment by Iamgoofball** [comment]
ehhhhhhhhhhhhhhhhh

**Comment by Cyberboss** [comment]
They think it don't be like it be

**Pull request closed without merging**